# An individual reach point
string id
geometry_msgs/Pose goal
bool reached
moveit_msgs/RobotState goal_state
moveit_msgs/RobotState seed_state
float64 score
